src/controls/Controls.spec.js

```diff
@@ -1 +1,58 @@
-// Test away!
\ No newline at end of file
+// Test away!
+import React from "react";
+import { render, fireEvent } from "react-testing-library";
+import "react-testing-library/cleanup-after-each";
+import Controls from "./Controls";
+
+describe("<Controls />", () => {
+  it("Is open and unlocked", () => {
```

Comment:
One of the hardest parts I find with testing is to write semantically correct tests.  The tests here is great, but I feel like the verbiage on what the test is actually testing can be improved.

Your test `Is open and unlocked` doesn't do the assertions justice.